pilot/docker/prepare_proxy_test.sh

```diff
@@ -1,238 +0,0 @@
-#!/bin/bash
```

Comment:
I think this was used once upon a time to verify that iptables are working as expected but it clearly rotted

cc @ayj - what do you think ?

Reply:
https://github.com/istio/istio/pull/3562 probably ought to include an equivalent unit test.

Reply:
Also, `prepare_proxy.sh` hasn't experienced much churn. I suspect this test isn't that far out of date.

Reply:
agreed, just that I noticed stuff like `bazel run...`